shopping_mall/backend/ai/agent/supervisor/executor.py

```
@@ -0,0 +1,442 @@
+                        _detect_order_action(user_message)
+                        if tc_obj.name == "call_order_agent"
+                        else _SUPERVISOR_TOOL_TO_INTENT.get(tc_obj.name, "other")
+                    ),
+                    success=True,
+                    latency_ms=latency_ms,
+                    empty_result=False,
+                    iteration=iteration + 1,
+                ))
+                logger.info(
+                    f"[supervisor] iter={iteration+1} tool={tc_obj.name} "
+                    f"latency={latency_ms}ms → {result_str[:120]}"
```

Ruff (0.15.10)

[warning] Logging statement uses f-string

(G004)